src/libaktualizr/uptane/isotpsecondary.h

```diff
@@ -10,17 +10,17 @@ class IsoTpSecondary : public SecondaryInterface {
  public:
   explicit IsoTpSecondary(const std::string& can_iface, uint16_t can_id);
 
-  EcuSerial getSerial() override;
-  HardwareIdentifier getHwId() override;
-  PublicKey getPublicKey() override;
+  EcuSerial getSerial() const override;
+  HardwareIdentifier getHwId() const override;
+  PublicKey getPublicKey() const override;
   bool putMetadata(const RawMetaPack& meta_pack) override;
-  int getRootVersion(bool director) override;
+  int getRootVersion(bool director) const override;
   bool putRoot(const std::string& root, bool director) override;
-  bool sendFirmware(const std::shared_ptr<std::string>& data) override;
-  Json::Value getManifest() override;
+  bool sendFirmware(const std::string& data) override;
+  Uptane::Manifest getManifest() const override;
 
  private:
-  IsoTpSendRecv conn;
+  mutable IsoTpSendRecv conn;
```

Comment:
Why is this `mutable` too?

Reply:
because most of methods in the secondary interface are 'const' now and isotpsecondary uses IsoTpSendRecv non-const methods from within these const methods implementation.